dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -168,6 +165,15 @@ public static void objectToXML ( Object obj, File f, boolean removeFirst ) {
         }
     }
 
+    public static void objectToXML(final Object obj, final OutputStream outputStream) {
+        if (xmlSerializer == null) {
```

Comment:
this should be encapsulated on a XMLSerializerUtil or similar such as
```
XMLSerializerUtil.getInstance().marshal(obj, xmlWriter);
```

Reply:
done https://github.com/dotCMS/core/pull/20173/commits/e491040a5cc382fa8a25457ba027679b4afa86b7#diff-827653c0f7910bf99c00088d8a5123e24ca2bf4288dff44aa8b54e067f1535b5R171